go/appencryption/pkg/persistence/dynamodb_test.go

```diff
@@ -172,7 +173,7 @@ func (suite *DynamoDBSuite) SetupTest() {
 	}
 	suite.putItemInDynamoDB(getDynamoDBItem(en, suite.instant))
 
-	suite.dynamodbMetastore = NewDynamoDBMetastore(suite.sess)
+	suite.dynamodbMetastore = NewDynamoDBMetastore(suite.sess, WithTableName(tableName))
```

Comment:
should we keep using the defaults (table name in this case) for test and just add additional tests for new functional options?

Reply:
Using a custom table name for the tests retains the current integration-level testing coverage we had before and we assert the custom table name (and table name default) works via other targeted unit tests below, so all bases should be covered.

Ideally unit tests wouldn't require external dependencies at all (such as a local dynamodb instance), but considering the current state of testing here I figured this approach would be a decent compromise.